go/libkb/env.go

```diff
@@ -517,6 +518,13 @@ func (e *Env) GetGregorDisabled() bool {
 	)
 }
 
+func (e *Env) GetGregorPingInterval() time.Duration {
+	return e.GetDuration(10*time.Second,
+		func() (time.Duration, bool) { return e.getEnvDuration("GREGOR_PING_INTERVAL") },
+		func() (time.Duration, bool) { return e.config.GetGregorPingInterval() },
+	)
```

Comment:
also can you check the command line?  To keep everything consistent everything can be configured in those 3 places.  Should be lots of examples to copy from....